specification/cloudshell/resource-manager/Microsoft.Portal/stable/2018-10-01/CloudShell.json

```diff
@@ -0,0 +1,952 @@
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Portal/locations/{location}/userSettings/{userSettingsName}": {
+      "get": {
+        "operationId": "getUserSettingsWithLocation",
+        "x-ms-examples": {
+          "GetUserSettings": {
+            "$ref": "./examples/GetUserSettings_Location.json"
+          }
+        },
+        "summary": "Get user settings.",
+        "description": "Get current user settings for current signed in user. This operation returns settings for the user's cloud shell preferences including preferred location, storage profile, shell type, font and size settings.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserSettingsNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GetUserSettingsResponse"
+            },
+            "headers": {
+              "x-ms-console-required-location-code": {
+                "type": "string",
+                "description": "normalized required location code"
+              },
+              "x-ms-console-preview-user": {
+                "type": "boolean",
+                "description": "indicates whether user is a previous usr."
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "patchUserSettingsWithLocation",
+        "summary": "patch user settings.",
+        "description": "Patch cloud shell settings for current signed in user",
+        "x-ms-examples": {
+          "PatchUserSettings": {
+            "$ref": "./examples/PatchUserSettings_Location.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserSettingsNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "name": "UserSettings",
+            "description": "The properties of the user settings to be updated.",
+            "required": true,
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/CloudShellPatchUserSettings"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
PATCH 200 response should include the same response body as the equivalent GET

Reply:
Fixed